[2023-12-21 10:00:04,255] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T09:00:00+00:00 [queued]>
[2023-12-21 10:00:04,305] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T09:00:00+00:00 [queued]>
[2023-12-21 10:00:04,306] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 10:00:04,306] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-21 10:00:04,307] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 10:00:04,325] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-21T09:00:00+00:00
[2023-12-21 10:00:04,340] {standard_task_runner.py:52} INFO - Started process 80362 to run task
[2023-12-21 10:00:04,350] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-21T09:00:00+00:00', '--job-id', '220', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmp6uf2g1tz', '--error-file', '/tmp/tmpbgqq7bx9']
[2023-12-21 10:00:04,354] {standard_task_runner.py:77} INFO - Job 220: Subtask fill
[2023-12-21 10:00:04,474] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T09:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-21 10:00:04,669] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-21T09:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-21T09:00:00+00:00
[2023-12-21 10:00:06,869] {python.py:118} INFO - Done. Returned value was: None
[2023-12-21 10:00:06,922] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231221T090000, start_date=20231221T100004, end_date=20231221T100006
[2023-12-21 10:00:07,050] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-21 10:00:07,099] {local_task_job.py:146} INFO - Task exited with return code 0
